### PR TITLE
[Performance Profile] - Specify flags in performance profile

### PR DESCRIPTION
**Pull request by smartdev09**
**Notes for Reviewers**

This PR fixes #

When i create a profile, subscribePerformanceProfiles() does not return metadata. When i reload the page fetchPerformanceProfiles() returns metadata and it is visible.

https://github.com/meshery/meshery/assets/31584129/7a9cdeba-1c63-4477-9d45-736839125444

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -38,12 +38,12 @@ require (
 	github.com/grafana-tools/sdk v0.0.0-20220919052116-6562121319fc
 	github.com/jarcoal/httpmock v1.3.0
 	github.com/jinzhu/copier v0.3.5
-	github.com/layer5io/gowrk2 v0.0.0-20191111234958-a4c9071c0f87
+	github.com/layer5io/gowrk2 v0.6.1
```
Comment (leecalcote) [inline go.mod:41]: @abdullah1308, will you confirm if we're going to have an issue with this newer version of gowrk2 and fortio?

Comment (leecalcote) [reply go.mod:41]: Ok, I see that this is @abdullah1308's commit. 👍

**Comment by UsmanDev09** [comment]
I have added a file upload option and it validates if it contains json or not. 

https://github.com/meshery/meshery/assets/31584129/24af9c80-98e8-409a-8dba-3668714ee1db

**Comment by MUzairS15** [comment]
@UsmanDev09 UI gets distorted when JSON is large, both at the time of upload and when viewing the profile. Try uploading a JSON with many options.

**Comment by MUzairS15** [comment]
@UsmanDev09 I have updated the subscription but could not verify, since after updating the subscription UI crashes if there is empty additional options.

**Comment by leecalcote** [comment]
I'll see about resolving these merge conflicts.

**Comment by UsmanDev09** [comment]
@MUzairS15 I have handled the large json file issue as follows: 


https://github.com/meshery/meshery/assets/31584129/e6ec3375-afaa-4e63-8ba5-3181acaa926d


I have also handled the empty additional option edge case.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
One last item @UsmanDev09, at time of uploading (a large JSON), will you also add expand / collapse ? It would be more better if expand / collapse is present at top itself, instead of need to scroll at bottom (to click on Show less)

**Comment by MUzairS15** [comment]
@UsmanDev09 Is it possible to add helper text for upload ssl certificate field, (to mention .crt files supported) ?

**Comment by UsmanDev09** [comment]
@MUzairS15 I have added `...` to indicate that full json is not being displayed. 

dots

**Comment by MUzairS15** [comment]
Screenshot 2023-08-02 at 6 45 01 PM

@UsmanDev09 A helper text in this form?

**Comment by UsmanDev09** [comment]
@MUzairS15 i have added the helper text and persisted the file name upon editing too. 

https://github.com/meshery/meshery/assets/31584129/3094e96d-10d1-43c2-96cb-abacb194e4f8

**Comment by UsmanDev09** [comment]
Upon uploading .crt files, the same subscription issue occurs. Reload is required to see the profile

**Review by MUzairS15 — DISMISSED** [review]
@UsmanDev09 Subscription is working for me, I can see updates without refreshing.
Thhough there is another issue, try editing a profile which already has a certificate, when you upload a new cert, the name field is not updated, instead label is updated. 
https://github.com/meshery/meshery/assets/79860504/bd55bd09-2294-40d5-8e1c-78c070b366dd

I am merging this PR, please handle this in separate PR.

**Pull request merged**